lib/fastifySession.js

```diff
@@ -18,8 +18,8 @@ function session (fastify, options, next) {
   fastify.decorate('decryptSession', (sessionId, request, callback) => {
     decryptSession(sessionId, options, request, callback)
   })
-  fastify.decorateRequest('sessionStore', options.store)
-  fastify.decorateRequest('session', {})
+  fastify.decorateRequest('sessionStore', { getter: () => options.store })
+  fastify.decorateRequest('session', { getter: () => ({}) })
```

Comment:
```suggestion
  fastify.decorateRequest('session', { getter: () => ({}) })
```
You cannot change it with `getter` only. Please change it to `null` and add a empty object before or on `preValidation` hook.

Reply:
Easy fix.
My bad.